app/pages/form-builder.js

```diff
@@ -1,6 +1,6 @@
 import React, {Component} from 'react';
 import {Container, Row, Col} from 'react-bootstrap';
-import Header from '../components/Header';
+// Import Header from '../components/Header';
```

Comment:
Remove this page? It's not used, and broken

Reply:
done